examples/parallel-optimization.ipynb

```diff
@@ -0,0 +1,205 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Parallel optimization\n",
+    "\n",
+    "Iaroslav Shcherbatyi, May 2017."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 29,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<matplotlib.figure.Figure at 0x7f363f37cfd0>"
+      ]
+     },
+     "execution_count": 0,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import numpy as np\n",
+    "np.random.seed(0)\n",
+    "\n",
+    "%matplotlib inline\n",
+    "import matplotlib.pyplot as plt\n",
+    "plt.set_cmap(\"viridis\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Problem statement\n",
+    "\n",
+    "In many practical cases, you can run evaluations of your objective in parallel, which you would be \n",
+    "great to leverage in order to speed up the black box optimization. \n",
+    " \n",
+    "Think about it like this: if you want to find a perfect mix of ingredients for a perfect cookie, you would rather \n",
```

Comment:
Another example can be distribute data across different jobs and running computations in parallel..?